src/Raven.Server/Documents/Handlers/Admin/RachisAdminHandler.cs

```diff
@@ -52,9 +52,25 @@ public Task GetClusterTopology()
             using(context.OpenReadTransaction())
             {
                 var topology = ServerStore.GetClusterTopology(context);
+                if (topology.Members.Count == 0)
+                {
+                    topology = new ClusterTopology(
+                        Guid.NewGuid().ToString(),
+                        null,
+                        new Dictionary<string, string>
+                        {
+                            ["A"] = HttpContext.Request.GetHostnameUrl()
```

Comment:
You need to get this value from the client. The hostname doesn't include port, etc.